src/garvis/heartbeat_cli.py

```
@@ -0,0 +1,76 @@
+
+def build_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(prog="garvis-heartbeat")
+    parser.add_argument("--root", type=Path, default=None)
+    sub = parser.add_subparsers(dest="command", required=True)
+    sub.add_parser("run-once")
+    sub.add_parser("daemon")
+    sub.add_parser("health")
+    return parser
+
+
+def main(argv: Sequence[str] = None) -> int:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)